account_credit_control/migrations/12.0.3.0.4/pre-migration.py

```diff
@@ -11,5 +11,9 @@ def migrate(env, version):
     This patch release fixes a bug whose result may be stored in database, so
     although it's still a patch, we need to clear the database, just in case.
     """
-    action = env.ref("account_credit_control.credit_control_communication_action")
-    action.domain = []
+    try:
+        action = env.ref("account_credit_control.credit_control_communication_action")
```

Comment:
Better use `, raise_if_not_found=False)` instead of try-except

Reply:
fixed, thanks :)